README.md

```diff
@@ -30,4 +30,70 @@ _If the aforementioned link says *ACCESS DENIED*, then contact me _[here](mailto
 
 * After raising a PR, and after setting up everything, you can click [here](https://dashboard.heroku.com/pipelines/45438218-0719-4c45-8d9f-50d610d24ed5/) and look at the tab called __REVIEW APPS__ and scroll down to find your PR and click on __Open App__ button.
 
-* Another easy way to check your PR's deployment, after TravisCI build passes, scroll down to the PR's page and click on the `View Deployment` button.
\ No newline at end of file
+* Another easy way to check your PR's deployment, after TravisCI build passes, scroll down to the PR's page and click on the `View Deployment` button.
+
+## Sequelize and Sequelize-cli
+
+This API uses Sequelize as its ORM. To get you started, we will configure sequelize using .sequelizerc file.
+
+The .sequelizerc file will in turn be used by sequelize-cli to setup the the sequelize folder structure.
+
+### 1. Configure environment variables
+
+The following are example environment variables needed:
+
+#### Database
+
+* DEV_DB_USERNAME=postgres
```

Comment:
you don't have to add values to those variables cause, for example, their username might be different from what you wrote.